x-pack/plugin/otel-data/src/main/resources/resources.yaml

```diff
@@ -0,0 +1,16 @@
+# "version" holds the version of the templates and ingest pipelines installed
+# by xpack-plugin otel-data. This must be increased whenever an existing template or
+# pipeline is changed, in order for it to be updated on Elasticsearch upgrade.
+version: 1
+
+component-templates:
```

Comment:
This is missing `traces@mappings` and `traces@settings` but these should be moved to the core anyway.